dev/com.ibm.ws.security.acme_fat/fat/src/com/ibm/ws/security/acme/docker/pebble/PebbleContainer.java

```diff
@@ -120,6 +123,16 @@ public void stop() {
 		super.stop();
 		network.close();
 	}
+	
+	public void stopChallengeServer() {
+		challtestsrv.stop();
+		Log.info(PebbleContainer.class, "stopChallengeServer", "Challenge server stopped.");
+	}
+	
+	public void startChallengeServer() {
```

Comment:
The same as stopChallengeServer().

Reply:
Update name and implementation, same as stop.